test/unit/logic/transactions/pool.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var async = require('async');
+var expect = require('chai').expect;
+
+var node = require('../../../node');
+
+var TransactionPool = require('../../../../logic/transactionPool');
+var TransactionLogic = require('../../../../logic/transaction');
+var TransferLogic = require('../../../../logic/transfer');
+var AccountModule = require('../../../../modules/accounts');
+var AccountLogic = require('../../../../logic/account');
+var modulesLoader = require('../../../common/initModule').modulesLoader;
+var transactionTypes = require('../../../../helpers/transactionTypes');
+
+describe('txPool', function () {
+
+	var txPool;
+
+	before(function (done) {
+		// Init transaction logic
+		async.auto({
+			accountLogic: function (cb) {
+				modulesLoader.initLogicWithDb(AccountLogic, cb);
```

Comment:
Please use `initLogic` instead of `initLogicWithDb` as database is not used in the test